satellite/metainfo/metainfo_test.go

```diff
@@ -336,6 +343,30 @@ func TestCommitSegment(t *testing.T) {
 			require.Error(t, err)
 			require.Contains(t, err.Error(), "Number of valid pieces is less than or equal to the repair threshold")
 		}
+		{
+			// error if number of remote pieces is lower then repair threshold
+			redundancy := &pb.RedundancyScheme{
```

Comment:
you need to test also rest of conditions from `validateRedundancy` method.

Reply:
added  below test cases 
			// error - ErasureShareSize <= 0
			// error - any of the values are negative
			// error - MinReq >= RepairThreshold
			// error - MinReq >= RepairThreshold
			// error - RepairThreshold >= SuccessThreshold
			// error -  SuccessThreshold >= Total
			// ok - valid RS parameters